pkg/scheduler/scheduler.go

```diff
@@ -203,7 +207,9 @@ func (s *Scheduler) assign(ctx context.Context, e *entry) {
 	go func() {
 		err := s.client.Update(ctx, newWorkload)
 		if err == nil {
-			log.V(2).Info("Successfully assigned capacity and resource flavors to workload")
+			msg := fmt.Sprintf("Successfully assigned %v to capacity %v", workload.Key(newWorkload), e.Capacity)
+			s.recorder.Eventf(newWorkload, corev1.EventTypeNormal, "Scheduled", msg)
+			log.V(2).Info(msg)
```

Comment:
We should use non-parameterized sentences in structured logging. The values are already in the logger, line 197.

Should we also use `klog.KObj` (or `KRef`, depending on the situation) for mentions of capacity and queue? It might help with querying logs. Split into #27